src/ontology/uberon-edit.obo

```diff
@@ -39,7 +39,6 @@ synonymtypedef: LATIN "latin term"
 synonymtypedef: MISSPELLING "a mis-spelling that is in common use and thus recorded"
 synonymtypedef: NON_AMNIOTE "synonym that is inappropriate for amniotes"
 synonymtypedef: NON_MAMMAL "synonym that is inappropriate for mammals"
-synonymtypedef: OMO:0003000 "abbreviation"
```

Comment:
@anitacaron this appears to cause an error `ERROR	missing_synonymtype_declaration	OMO:0003000	rdfs:subPropertyOf	`

Reply:
Yes, but the definition is now in the `merged_imports.owl` and should not have duplications. We'll probably lower the severity of this QC.